helpers/check_ip_in_list.js

```diff
@@ -26,51 +26,22 @@ var ip = require('ip');
  * @param {boolean} returnListIsEmpty - The return value, if list is empty
  * @returns {boolean} True if ip is in the list, false otherwise
  */
-function CheckIpInList(list, addr, returnListIsEmpty) {
+function CheckIpInList(list, addr) {
 	var i;
```

Comment:
more descriptive variable naming here?

Reply:
dunno, it is a simple iterator. Do you have suggestions?

Reply:
Then we should just declare it in the for loop like: `for(let i =0; i < list.length; i++)`